donut/modules/core/routes.py

```diff
@@ -29,3 +29,12 @@ def get_members_list():
 def get_members(user_id):
     """GET /1/members/<int:user_id>/"""
     return json.dumps(helpers.get_member_data(user_id))
+
+
+@blueprint.route("/1/members/<int:user_id>/groups/")
+def get_group_list_of_member(user_id):
+    '''- /1/members/3/groups/ group_members table grabbing the groups
```

Comment:
You can remove the leading `-`.
Also maybe just keep `/1/members/3/groups/`, and the part that says `List all groups that a member is in`